pomegranate/hmm.pyx

```diff
@@ -2486,8 +2490,10 @@ cdef class HiddenMarkovModel(GraphModel):
                     initial_log_probability_sum = log_probability_sum
                 else:
                     improvement = log_probability_sum - last_log_probability_sum
+                    time_spent = time.time() - epoch_start_time
                     if verbose:
-                        print("Training improvement: {}".format(improvement))
+                        print("Training improvement: {} in {:.2f}s".format(improvement,
+                                                                           time_spent))
```

Comment:
@jmschrei Just a quick note with regards to #156 -- the alignment of `improvement` and `time_spent` here is possible outright due to the use of spaces. If spaces were not used, the first part of the line (up until `print`) would have to be indented with tabs and from then on spaces could be used. This would make it quite difficult to put together.